publishing-rules.yaml

```diff
@@ -54,19 +48,19 @@ rules:
       dir: vendor/github.com/containers/image
 - destination: opencontainers-runc
   branches:
-  - name: openshift-3.9
+  - name: openshift-3.10-runc-595bea0
```

Comment:
i would rather leave the 3.9 branch there so OSE 3.9 or origin 3.9 still have the right level and just add 3.10 branches.